Initial commit with full core functionalities for GEE extractor

marcgarnica13 opened the pull request:
- run.py controls the process execution and its arguments.
 - Arguments can be passed by command line or by a conf file by
specifying its path.
 - Full script API is going to be document in the repository README.

marcgarnica13 committed:
Initial commit with full core funcitonalities for GEE extractor

 - run.py controls the process execution and its arguments.
 - Arguments can be passed by command line or by a conf file by
specifying its path.
 - Full script API is going to be document in the repository README.

marcgarnica13 committed:
Updating DOC and adding DHIS2API from ESTools

 - dhis2api.py from ESTools integrated into the script.
 - Updating README.md file with context, usage and basic installation.

marcgarnica13 committed:
Adding Usage syntax to README file

ifoche reviewed (changes requested):
I would include at least the following in the README
- The signup process here...
https://signup.earthengine.google.com/#!/
- The execution of the command the first time running the script
```
earthengine authenticate
```

Maybe we need to review also the requirements.txt file?

marcgarnica13 committed:
Full purge of requirements.txt and installation from scratch

marcgarnica13 committed:
Updated README.md with Google Earth authentication steps

marcgarnica13 commented:
- Updated requirements deleted unused (imported by default by conda) packages.
 - Adding Google Earth Engine authenticating steps in the README.md file basic installation.

marcgarnica13 committed:
Output message for import execution

 - Adding three different output for each orgunit present in the import:
   - SUCCESS: Import done + name,id and period.
   - FAIL (1): Organistion unit skipped to the lack of coordinates in
the DHIS2 instance.
   - FAIL (2): Organisation unit skipped due to geometry sizes. Higher
level organisation units in the orgUnit tree will not contain directly
exported values from E.E but gather them from their subtree orgunits.
